app/controllers/home_controller.rb

```
@@ -20,28 +29,94 @@ def logout
+        respond_to do |format|    
+            format.json { render json: {errors: {password_confirmation: "password confirmation required!"}}, status: :unprocessable_entity  }
+        end
+      end
+      
+    else
+      respond_to do |format|
+        format.json { render json: {errors: {old_password: "doesn't match!"}}, status: :unprocessable_entity }      
+      end
+    end
+    
+  end
```

RuboCop (1.76.1)

[convention] Assignment Branch Condition size for `verify_change_password` is too high. [<7, 24, 7> 25.96/23]

(Metrics/AbcSize)